.github/workflows/ci.yml

```diff
@@ -488,12 +488,9 @@ jobs:
       uses: actions/setup-python@v5.0.0
       with:
         python-version: ${{ matrix.PYTHON }}
-    - uses: actions-rs/toolchain@v1
```

Comment:
@reaperhulk

Reply:
This change should be fine. The rust-toolchain@rev behavior is a bit unfortunate for pinning purposes. You can still pin and use `toolchain`, but dependabot won't bump it automatically.